api/nodes/serializers.py

```diff
@@ -274,7 +273,6 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
     tags = ValuesListField(attr_name='name', child=ser.CharField(), required=False)
     access_requests_enabled = ShowIfVersion(ser.BooleanField(read_only=False, required=False), min_version='2.0', max_version='2.8')
     node_license = NodeLicenseSerializer(required=False, source='license')
-    analytics_key = ShowIfAdminScopeOrAnonymous(ser.CharField(read_only=True, source='keenio_read_key'))
```

Comment:
Per @brianjgeiger's comment, we can set this to `''` rather than delete it. If we do that, we don't need to worry about versioning. Maybe add a short comment saying that this can be removed after another version release.